app/models/fe/state_chooser.rb

```diff
@@ -3,8 +3,8 @@
 # - drop down of states
 module Fe
   class StateChooser < Question
-    def choices(country = 'US')
-      @states = Carmen.states(country)
+    def choices(locale = nil, country = 'US')
```

Comment:
After this change, if i call `choices('US')` does it assume that 'US' is the locale or the country?